tests/pre_merge/tools/test_gradio_entrypoint.py

```diff
@@ -0,0 +1,88 @@
+"""Test Gradio inference entrypoint script."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from __future__ import annotations
+
+import sys
+from importlib.util import find_spec
+from unittest.mock import patch
+
+import pytest
+
+from anomalib.data import TaskType
+from anomalib.deploy import ExportMode, OpenVINOInferencer, TorchInferencer, export
+from anomalib.models import get_model
+
+sys.path.append("tools/inference")
+
+
+@pytest.mark.order(5)
+class TestGradioInferenceEntrypoint:
+    """This tests whether the entrypoints run without errors without quantitative measure of the outputs.
+
+    Note: This does not launch the gradio server. It only checks if the right inferencer is called.
+    """
+
+    @pytest.fixture
+    def get_functions(self):
+        """Get functions from Gradio_inference.py"""
+        if find_spec("gradio_inference") is not None:
+            from tools.inference.gradio_inference import get_inferencer, get_parser
+        else:
+            raise Exception("Unable to import gradio_inference.py for testing")
+        return get_parser, get_inferencer
+
+    def test_gradio_inference(self, get_functions, get_config, project_path, transforms_config):
```

Comment:
Would it be possible to split this function to improve readability? For example, `test_gradio_torch_inference`, `test_gradio_openvino_inference` or something similar?